tests/projects/test_views.py

```diff
@@ -44,6 +47,58 @@ def test_project_create(app: DjangoTestApp):
     assert Version.objects.get_for_object(added_project.first()).first().revision.comment == Project.CREATED
     assert FilerImage.objects.count() == 1
     assert added_project.first().image.original_filename == "example.png"
+    assert not added_project.first().organization
+
+def test_project_create_with_organization(app: DjangoTestApp):
+    user = UserFactory()
+    app.set_user(user)
+    organization = OrganizationFactory()
+    ct = ContentType.objects.get_for_model(organization)
+    RepresentativeFactory(
+        content_type=ct,
+        object_id=organization.pk,
+        user=user
+    )
+
+    form = app.get(reverse("project-create")).forms['project-form']
+    form['title'] = "Project"
+    form['description'] = "Description"
+    form['organization'] = organization.id
+    form['url'] = "example.com"
+    form['image'] = Upload('example.png', generate_photo_file(), 'image')
+    resp = form.submit()
+
+    added_project = Project.objects.filter(title='Project')
+    assert added_project.exists()
+    added_project = added_project.first()
```

Comment:
`QuerySet.first()` returns first object in queryset or None if queryset has no objects. So this code can be simplified a bit:

```suggestion
    added_project = Project.objects.filter(title='Project').first()
    assert added_project
```